sections/04_study.md

```diff
@@ -59,18 +59,52 @@ particularly notable in this area?*
 *There is not much content here.  Can [@tag:Chen2015_trans_species] be covered
 elsewhere?*
 
-### Cellular phenotypes
-
-*These are primarily imaging-based phenotypes.  We have not surveyed this area
-very comprehensively.  We could decide to not make imaging a primary focus,
-refer to existing reviews, and mention only a few particularly noteworthy
-representative papers.  Alternatively, we need to expand our literature review
-and summaries immediately if someone wants to be responsible for this
-sub-section.*
-
-*Transfer learning from non-biological datasets to biological imaging
-data could fit here, and that does seem like an important topic.  Or
-transfer learning could be a more general topic for the Discussion section.*
+### Morphological phenotypes
+
+A field poised for dramatic revolution by deep learning is
+bioimage analysis - and not in the way you might first think.
```

Comment:
I'm a bit confused - what is the way that I might at first think? Is the point that:
* Deep learning + images is generally supervised.
* However sometimes we need to go fishing.
* Unsupervised deep learning approaches provide the opportunity to identify features that we don't know we should be looking for - this will be the revolution (and what is unexpected)?

If this is the idea, I'd suggest leading the reader through it a bit more. Maybe by first highlight how deep learning for supervised tasks has been great. Then come in later with - but, hey, this unsupervised stuff is the real game changer.

If this isn't the idea, then it went over my head. Maybe a bit more of a lead through the section would be helpful. I like the conversational tone. We may change it a bit later as we integrate the sections, but we'll probably also revise things that are less conversational to be more so.